testing/bdd/conftest.py

```diff
@@ -133,12 +133,26 @@ def gen_second_document_fixture(test_context, template_dict, template_file_two):
 
 @when('the EBU-TT-Live document is denested')
 def convert_to_ebuttd(test_context):
+    print('denester input')
+    print(test_context["document"].get_xml())
```

Comment:
Did you mean to keep all these `print` statements in? I like them because they only show up when a test fails, so feel free to keep them in.

Reply:
Yes I did, for the same reason.